### ISSUE TITLE
Create a logo

### ISSUE THREAD

**Issue by cburgdorf** [nickel-org/nickel.rs#43]
Maybe something inspired by the metal. Could also be inspired by the coin but I think I like the shape of a metal more.

Some inspiration:
https://www.google.de/search?q=nickel&safe=off&es_sm=91&source=lnms&tbm=isch&sa=X&ei=NQ7bU6WAA_Ha0QXprIGgDg&ved=0CAgQ_AUoAQ&biw=1440&bih=722

**Comment by cburgdorf** [comment]
Would love to revisit that and get @PascalPrecht involved.

**Comment by PascalPrecht** [comment]
Maybe @timche wants to contribute?

**Comment by timche** [comment]
Yeah, why not. I will have a look on all the comments and see what I can do.

**Comment by PascalPrecht** [comment]
@timche ping me if you need/want a one-on-one vc

**Comment by clintnetwork** [comment]
Gotham.rs have a pretty good logo, maybe something like this but with a nickel roch
